mpas_analysis/test/test_mpas_xarray.py

```diff
@@ -41,6 +41,15 @@ def test_subset_variables(self):
                                                              yearoffset=1850))
         self.assertEqual(ds.data_vars.keys(), varList)
 
+        with self.assertRaisesRegexp(AssertionError, 'Empty dataset is returned.'):
```

Comment:
I found more info in the docs here:
https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaisesRegexp
Sorry to bother you about it.  I think I get it now.